### DISCUSSION TITLE
Permanent View-ICS table

### DISCUSSION THREAD

**Discussion by ghost** [the-djmaze/snappymail#1662] [General]
Hey is it possible to have a permanent view of the table in the view-ics plugin. 
Normally it is collapsed and users need to press a button, but our users aren't really able to understand that :O so it would be very nice to have an option for a permanent view of this table without user action needed.
I tried to change it via css and also tried to change it in message.js and message.min.js but I am not able to get it done by myself.
<img width="363" alt="Bildschirmfoto 2024-07-11 um 08 39 42" src="https://github.com/the-djmaze/snappymail/assets/76497172/23fc9393-a7a3-4067-b7d7-b5ba55286f0d">

![Bildschirmfoto 2024-07-11 um 08 37 30](https://github.com/the-djmaze/snappymail/assets/76497172/9733269d-46d1-4756-a6a1-39b6a243a68d)

**Comment by ghost** [comment]
@the-djmaze Hey is there any possible way to solve my problem?

**Reply by the-djmaze** [reply]
Not at the moment, but maybe i can add something to the plugin

**Reply by ghost** [reply]
That would be wonderful, thanks
